pages/linux/eww.md

```diff
@@ -0,0 +1,28 @@
+# eww
+
+> Implement your own custom widgets.
+> More information: <https://elkowar.github.io/eww>.
+
+- Start the daemon:
+
+`eww daemon`
+
+- Open a widget:
+
+`eww open {{/path/to/directory}}`
+
+- Close a widget:
+
+`eww close {{/path/to/directory}}`
```

Comment:
Is the specified directory the directory where the source code for the widget is located? This should be made explicit in my opinion.